lib/manager/gradle/gradle-updates-report.ts

```diff
@@ -46,11 +48,19 @@ allprojects {
            .findAll { !it.startsWith('file:') }
            .unique()
         project.repositories = repos
-        def deps = (buildscript.configurations + configurations)
-          .collect { it.dependencies + it.dependencyConstraints }
-          .flatten()
-          .findAll { it instanceof DefaultExternalModuleDependency || it instanceof DependencyConstraint }
-          .collect { ['name':it.name, 'group':it.group, 'version':it.version] }
+        def deps = []
+        (buildscript.configurations + configurations)
+           .collect {
+               def name = it.name == 'classpath' ? 'plugins' : it.name
```

Comment:
I’m not sure that changing `classpath` to `plugins` is a good idea. The configuration _is_ called `classpath` in gradle (meaning the classpath of Gradle itself).

As a gradle user, if we document this feature as “you can select the depency updates by configuration name”, I would expect that I can select plugin dependencies via “classpath”.

Note that you can also declare classpath dependencies this way:

```kotlin
buildscript {
    dependencies {
      classpath(…)
    }
}
```

In this case, it would be especially irritating that the configuration is all of a sudden called `plugins`

Reply:
The main reason was to make it easier for people to find which configuration they actually need to refer to for plugins. But I agree, it is quite a dirty solution, probably is easier to just have some examples around this.

Reply:
> have some examples around this

Yes, I think having some examples in the docs would be helpful.